rclcpp_components/src/component_manager.cpp

```diff
@@ -179,6 +179,17 @@ ComponentManager::OnLoadNode(
           }
           options.use_intra_process_comms(extra_argument.get_value<bool>());
         }
+
+        // --params-file support
+        if (extra_argument.get_name() == "--params-file") {
```

Comment:
Why this argument is prefixed with `--` and the one above (`use_intra_process_comms`) isn't.
That sounds weird to me.

Reply:
I didn't notice and it worked fine. I'll try to remove the '--' and see if it still works.

Regardless, I suppose I should move it to a const since it's referenced more than once... :)